cibuild.cmd

```diff
@@ -156,6 +156,20 @@ if %ERRORLEVEL% NEQ 0 (
     goto :error
 )
 
+:: Only detect source control changes when running in the CI environment
+:: Detect if there are any changed files which should fail the build
+if DEFINED JENKINS_URL (
+    echo Detecting changed files...
+    git diff --cached HEAD --exit-code
```

Comment:
I'm no git expert, but it worked when I tested it.  I'm going to try this instead:

`git --no-pager diff HEAD --word-diff=plain --exit-code` which seems to work...